signup.js

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
```

Comment:
this should be a .jsx file, also move this to the components folder and just put it in the authentication folder, you won't have merge conflicts

Reply:
this matters bc the linting only runs on jsx files in components folder